plugins/mongodb/mongodbembed.hpp

```diff
@@ -227,6 +231,7 @@ namespace mongodbembed
             embeddedScript = end + 1; // Save embedded script as what is after the parenthesis
             if(embeddedScript.find(";") == std::string::npos) failx("Syntax Error: missing semicolon");
             cursor = embeddedScript.c_str();
+            queryString.append(embeddedScript.c_str());
```

Comment:
why don't we perform a queryString.set rather than append here?

Reply:
That was an oversight. I was not familiar with the StringBuffer methods. I have changed it to use set().

Reply:
Not a big deal, in fact I think it's fairly common in our code base to use the append method as long as we can ensure the string is empty, just wanted to make sure we were doing what we intended to do.